ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh

```diff
@@ -517,6 +517,11 @@ echo "Promtail manifests created, the cluster can be found at https://grafana-lo
 if [[ -f "/usr/bin/python3" ]]; then
   ENCODED_INVOKER="$(python3 -c "import urllib.parse; print(urllib.parse.quote('${OPENSHIFT_INSTALL_INVOKER}'))")"
   cat >> ${SHARED_DIR}/custom-links.txt << EOF
-  <a target="_blank" href="https://grafana-loki.ci.openshift.org/explore?orgId=1&left=%5B%22now-24h%22,%22now%22,%22Grafana%20Cloud%22,%7B%22expr%22:%22%7Binvoker%3D%5C%22${ENCODED_INVOKER}%5C%22%7D%20%7C%20unpack%22%7D%5D">Loki</a>&nbsp;<a target="_blank" href="https://hackmd.io/zx5Av4mTQKqdL7EF0JyYBQ?view">Loki cheat sheet</a>
+  <p>
+      <a target="_blank" href="https://grafana-loki.ci.openshift.org/explore?orgId=1&left=%5B%22now-24h%22,%22now%22,%22Grafana%20Cloud%22,%7B%22expr%22:%22%7Binvoker%3D%5C%22${ENCODED_INVOKER}%5C%22%7D%20%7C%20unpack%22%7D%5D">Loki</a> - Loki provides searchable log information for all containers that existed on a cluster, not just those present at the end of the run like must-gather.
+  </p>
+  <p>
+      <a target="_blank" href="https://hackmd.io/zx5Av4mTQKqdL7EF0JyYBQ?view">Loki cheat sheet</a> - Common expressions used to search Loki log results.
```

Comment:
```suggestion
      <a target="_blank" href="https://gist.github.com/vrutkovs/ef7cc9bca50f5f49d7eab831e3f082d8">Loki cheat sheet</a> - Common expressions used to search Loki log results.
```
to have blocked CORS fixed (#20930)